java/code/src/com/redhat/rhn/frontend/xmlrpc/configchannel/ConfigChannelHandler.java

```diff
@@ -279,6 +281,33 @@ public ConfigRevision getFileRevision(User loggedInUser, String configChannelLab
         return cm.lookupConfigRevisionByRevId(loggedInUser, cf, revision.longValue());
     }
 
+    /**
+     * Restore missing salt files on the disk based on the current state of the database.
+     * @param loggedInUser The current user
+     * @param channelLabels the list of global channels.
+     * @return 1 if successful with the operation, errors out otherwise.
+     *
+     * @xmlrpc.doc Restore missing salt files on the disk based on the current state of the database.
```

Comment:
this test will be on docs for API. I would put something like "Synchronize all files on the disk to the current state of the database"